src/peer_connection.cpp

```diff
@@ -4872,7 +4872,7 @@ namespace libtorrent {
 			&& m_reading_bytes == 0
 			&& !m_choked
 			&& m_peer_interested
-			&& t && t->is_upload_only()
```

Comment:
where is ``t`` checked?

Reply:
https://github.com/arvidn/libtorrent/blob/88643dd121d8d06c805fefae20216e346b2f9794/src/peer_connection.cpp#L4740

Reply:
ah, line 4740